### PR TITLE
Adding sticker and reaction message types.

### PR DESCRIPTION
**Pull request by Shamkh**
# Choose a PR Template
Design document: https://microsoft-my.sharepoint-df.com/:w:/p/shamkh/EQBw9SjIurVHvk_WF1u6L50B8E9vk_0D31mD-7Wry5TdCw?e=taPYNeb 
Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/communication/Communication.Messages/models.tsp [hunk 1]
```diff
@@ -205,6 +213,28 @@ model AudioNotificationContent extends NotificationContent {
...
+@doc("A request to send a Reaction notification.")
+model ReactionNotificationContent extends NotificationContent {
+  @doc("Message notification type is reaction.")
+  kind: CommunicationMessageKind.reaction;
+
+  @doc("emoji content like \\uD83D\\uDE00.")
+  emoji?: string;
+
+  @doc("ID of the previous message you want to reply to.")
+  messageId?: string;
```
Comment (weidongxu-microsoft) [inline specification/communication/Communication.Messages/models.tsp:226]: nit, why is `emoji` optional?

What happens when user send this without `emoji` nor `messageId`?

**Review by weidongxu-microsoft — COMMENTED** [review]
Remind, you need to merge when this whole api-version is ready.

You should not merge e.g. sticker/reaction first into it; afterward merge interaction into this same api-version.

File: specification/communication/Communication.Messages/models.tsp (hunk 1 above)
Comment (Shamkh) [reply specification/communication/Communication.Messages/models.tsp:226]: Updating this.

**Comment by Shamkh** [comment]
> Remind, you need to merge when this whole api-version is ready.
> 
> You should not merge e.g. sticker/reaction first into it; afterward merge interaction into this same api-version.

**Comment by Shamkh** [comment]
Closing this PR as we decided for preview version for Inetractive message type and these 2 types reaction and Sticker are also going together.

**Pull request closed without merging**